### PR TITLE
Cactus 903  tooltip remove position wrapper

### PR DESCRIPTION
**Pull request by maxrcollins**
Ticket: https://repayonline.atlassian.net/browse/CACTUS-903

UAT:

- Try adding the tooltip position function that I left in the mdx file to the tooltip in the storybook docs and see how it behaves.

### CODE REVIEW

**Review by daniloPenaR — APPROVED** [review]
LGTM

File: modules/cactus-web/src/Tooltip/Tooltip.mdx [hunk 1]
```diff
@@ -27,6 +27,21 @@ The `Tooltip` component can be used to provide helpful tips to end users.
...
+
+Example position function function:
+```jsx
+(tooltip, trigger) => {
+  const triggerRect = trigger.getBoundingClientRect()
+  tooltip.style['top'] = `25px`
+  tooltip.style['left'] = `${triggerRect.right}px`
+  tooltip.style['borderRadius'] = '0px'
+}
+```
```
Comment (maxrcollins) [inline modules/cactus-web/src/Tooltip/Tooltip.mdx:43]: i should probably put this below the `basic usage` section

**Review by wilysword — APPROVED** [review]

**Comment by wilysword** [comment]
At least one of your commits should have a `BREAKING CHANGE:` section. All of them should be formatted like this (or somthing along these lines):
```
feat(cactus-web): remove wrapper function from tooltip position prop

BREAKING CHANGE: call signature of position prop is changed from `(Rect, Rect) => Styles` to `(Element, Element) => void`

CACTUS-903
```

**Comment by maxrcollins** [comment]
> At least one of your commits should have a `BREAKING CHANGE:` section. All of them should be formatted like this (or somthing along these lines):


Ah sorry. I should've gone back and checked how we've named these in the past. I'll do that next time

**Pull request merged**